src/biogeochem/VOCEmissionMod.F90

```diff
@@ -933,13 +966,30 @@ function get_gamma_T(t_veg240_in, t_veg24_in,t_veg_in, ct1_in, ct2_in, betaT_in,
     if ( (t_veg240_in > 0.0_r8) .and. (t_veg240_in < 1.e30_r8) ) then 
        ! topt and Eopt from eq 8 and 9:
        topt = co1 + (co2 * (t_veg240_in-tstd0))
-       Eopt = Ceo_in * exp (co4 * (t_veg24_in-tstd0)) * exp(co4 * (t_veg240_in -tstd0))
+       if ( (ivt_in == nbrdlf_dcd_brl_shrub) ) then  ! boreal-deciduous-shrub
+       ! coming from BEAR-oNS campaign willows results
+             Eopt = 7.9 * exp (0.217_r8 * (t_veg24_in-273.15_r8-24.0_r8))
+       else if ( (ivt_in == nc3_arctic_grass ) ) then  ! boreal-grass
+             Eopt = exp(0.12*(t_veg240_in-288.15_r8))
+       else
+             Eopt = Ceo_in * exp (co4 * (t_veg24_in-tstd0)) * exp(co4 * (t_veg240_in -tstd0))
+       endif
+
     else
        topt = topt_fix
        Eopt = Eopt_fix
     endif
     x = ( (1._r8/topt) - (1._r8/(t_veg_in)) ) / ct3
-    gamma_t_LDF = Eopt * ( ct2_in * exp(ct1_in * x)/(ct2_in - ct1_in * (1._r8 - exp(ct2_in * x))) )
+    ! for the boreal grass from BEAR-oNS campaign
+    if ( (ivt_in == nc3_arctic_grass ) ) then  ! boreal-grass
+        bet_arc_c3  = 95+9.49*exp(0.53*(288.15_r8-t_veg240_in))
+        if (bet_arc_c3 .gt. 300) then
+            bet_arc_c3 = 300
+        endif
+        gamma_t_LDF = Eopt * exp(bet_arc_c3*((1/303.15_r8 - 1.0_r8/(t_veg_in))/ct3))
```

Comment:
We do not mix integers and reals, so in "real" calculations (or assignments and if-statements as above) we need integers to be real.

Also we need all hardwired real values to be followed by _r8.